presto-docs/src/main/sphinx/sql/explain.rst

```diff
@@ -44,6 +44,9 @@ distributed between fragments:
 Examples
 --------
 
+EXPLAIN (TYPE LOGICAL)
+^^^^^^^^^^^^^^^^^^^^^^
+
 Logical plan:
```

Comment:
Let’s remove these redundant descriptions now

Reply:
I kept them consciously, since the are kind of semi-sentence. The header is not.